src/style.css

```diff
@@ -1,3 +1,65 @@
+:root {
+  --active-color: #00acdc;
+}
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+}
+
 body {
+  background-color: #e5e5e5;
+  margin: 0;
+}
+
+.header {
+  background-color: white;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  position: relative;
+  padding: 0 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
```

Comment:
The `.logo` class has fixed dimensions. Consider using responsive units or media queries to ensure the logo scales appropriately on different screen sizes.